Makefile

```diff
@@ -34,12 +50,22 @@ api:
 	-w=/api/ custom-protoc sh -c "buf generate && \
 		buf lint && \
 		buf breaking --against 'buf.build/project-kessel/relations-api' "
-		
+
 
 .PHONY: build
 # build
 build:
-	mkdir -p bin/ && go build -ldflags "-X main.Version=$(VERSION)" -o ./bin/ ./...
+	$(warning Setting GOEXPERIMENT=strictfipsruntime,boringcrypto - this generally causes builds to fail unless building inside the provided Dockerfile. If building locally, run `make local-build`)
```

Comment:
This breaks the existing `make pr-check` target that I run locally on PRs, can you update it to `make local-build` instead of `make build`. Other than that it looks good 👍

Reply:
Oh and the `make run` target as well is targeting the old `build` target

Reply:
Targets updated, thank you!